source/computer/linux101.rst

```diff
@@ -1,2 +1,54 @@
 Linux 入门
 ==========
+
+:本节贡献者:
+:最近更新日期:
+:预计花费时间:
+
+----
+
+命令行
+-------
+
+命令行与图形界面
+^^^^^^^^^^^^^^^^
+
+终端
+^^^^^
+
+文件管理
+---------
+
+熟悉 Linux 文件系统
+^^^^^^^^^^^^^^^^^^^
+
+操作文件与目录
+^^^^^^^^^^^^^^
+
+文件路径
+^^^^^^^^^
+
+文件权限
+^^^^^^^^
+
+Shell 基础
+----------
+
+重定向
+^^^^^^
+
+管道
+^^^^
+
+通配符
+^^^^^^
+
+正则表达式
+^^^^^^^^^^
+
```

Comment:
```suggestion

Shell 脚本
------------

```

Reply:
Not sure if we really want to talk about shell scripts here. We have a section for bash in the programming chapter.

Reply:
Yes, I forgot that. Let's remove it.